win32ss/user/ntuser/caret.c

```diff
@@ -12,8 +12,8 @@ DBG_DEFAULT_CHANNEL(UserCaret);
 
 /* DEFINES *****************************************************************/
 
-#define MIN_CARETBLINKRATE 100
-#define MAX_CARETBLINKRATE 10000
+#define MIN_CARETBLINKRATE 200
```

Comment:
```suggestion
```
Not used anymore. But maybe those defines could be used in the CPL files that you have modified in this PR.